mod-orgs/schemas/account.json

```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "A vendor account",
```

Comment:
This one I'm not sure about, but to be on the safe side, please update this description to use "organization" instead of "vendor"